client/hotrod-client/src/test/java/org/infinispan/client/hotrod/impl/iteration/MediaTypeMultiServerRemoteIteratorTest.java

```diff
@@ -29,6 +29,7 @@ protected void createCacheManagers() throws Throwable {
    private ConfigurationBuilder getCacheConfiguration() {
       ConfigurationBuilder builder = hotRodCacheConfiguration(getDefaultClusteredCacheConfig(CacheMode.DIST_SYNC, false));
       if (key != null) builder.encoding().key().mediaType(key);
+      if (key != null) builder.encoding().value().mediaType(key);
```

Comment:
Shouldn't this be the value media type being passed?

Reply:
Let me mix the key and value types and see if there is an issue.

Reply:
Any time the value is configured with `text/plain,` the test fails. I can take a closer look to see why.